docs/api-reference/deckgl.md

```diff
@@ -75,9 +75,13 @@ This is helpful when rendered objects are difficult to target, for example
 irregularly shaped icons, small moving circles or interaction by touch.
 Default `0`.
 
-##### `pixelRatio` (Number, optional)
```

Comment:
> `pixelRation` is not used anywhere.

Maybe do a quick investigation and figure out when this was dropped. There is no mention in what's new or upgrade guides about this...